gammapy/data/observations.py

```diff
@@ -35,6 +35,9 @@ class Observation:
         Point spread function
     bkg : `~gammapy.irf.Background3D`
         Background rate model
+    rad_max: `~gammapy.irf.RadMax2D` or `~astropy.units.Quantity`
+        Only for point-like IRFs: RAD_MAX table (energy dependent RAD_MAX)
+        or a single angle (global RAD_MAX)
```

Comment:
Will this work with either a `RadMax2D` object and a single quantity? 

I would expect that at some point one would need to use the `RadMax2D.evaluate()` method.  This won't work with a `Quantity` object.

Reply:
In particular, in the current scheme there is no HDU  for the RAD_MAX in 1D. It is not clear to me how one can identify that it exists.
What would happen if you were to open one of joint Crab data release?

Reply:
@registerrier I don't understand the direction you are going with the second question. AFAIK none of the open crab data sets have a RAD MAX 2D table, so `rad_max` will be `None` for those.

Reply:
> In particular, in the current scheme there is no HDU for the RAD_MAX in 1D.

I don't understand this sentence. What do you mean with "1D" in here?